ceph/rados/core_workflows.py

```diff
@@ -428,7 +428,7 @@ def bench_write(self, pool_name: str, **kwargs) -> bool:
         Returns: True -> pass, False -> fail
         """
         duration = kwargs.get("rados_write_duration", 200)
-        byte_size = kwargs.get("byte_size", 4096)
+        byte_size = str(kwargs.get("byte_size", 4096)).upper()
```

Comment:
Converting number to upper case ?

Reply:
No, it is being type-casted to String

Reply:
Vaibhav, the reason behind this code change is a change in `radosbench` behaviour where Object size or Byte size having a suffix "Kb" previously used to be considered as KBytes but now complains "Illegal prefix"
Refer - https://bugzilla.redhat.com/show_bug.cgi?id=2348369#c1

For radosbench:
4096 -> 4096 bytes
4096K -> 4096 KBytes
4096KB -> 4096 Kbytes
4096 Kb -> used to work as 4096 Kbytes but no longer works this way, hence this change to explicitly type cast the input to uppercase

The default value is indeed a int, which is then being type cast to upper case string, in most of the cases it is expected that the user will send the `byte_size` argument with a suffix KB/MB or whatever.